README.md

```diff
@@ -149,6 +151,16 @@ Props marked with `*` are **mandatory**:
 | enabledNavigationWidget      | Show or hide navigation widget. Defaults to `false`                                                                                   |
 | isHighestLowestMarkerEnabled | Show or hide the highest and lowest tick on the chart. Defaults to `false`.                                                           |
 
+#### InitialData
+
+Initial data property designed to pass prepared data chart in case you don't want to use Feed or you want to make chart boost for initial. It get below properties and all of them are optional, so if you pass data to them, it cause the chart to use that data, and if you pass `null` it cause the chart to use the Feed call and get api.
```

Comment:
```suggestion
Initial data property designed to pass prepared chart data in case you don't want to wait for Feed data or if you simply want to make the chart render quicker on its initial load. It gets the properties below and all of them are optional, so if you pass the data, the chart will use that data instead, but if you pass `null` instead, the chart will default to the Feed call and get the data from API.
```